maintainers/maintainer-list.nix

```diff
@@ -2228,6 +2228,15 @@
     email = "tkatchev@gmail.com";
     name = "Ivan Tkatchev";
   };
+  ivanbrennan = {
```

Comment:
The addition to the maintainers list should be in a separate commit

Reply:
Makes sense. I'll split it out to its own commit.